src/v/pandaproxy/rest/handlers.cc

```diff
@@ -9,6 +9,7 @@
 
 #include "handlers.h"
 
+#include "config/rest_authn_endpoint.h"
```

Comment:
Is this include needed?

Reply:
Yes, to access authn_type. but I'll take another look.

Reply:
Oh, right, I guessed I missed something.